cypress/commons/utils/apiUtils.js

```diff
@@ -294,6 +295,16 @@ const interceptGetDatasets = () => {
   return alias;
 };
 
+const interceptGetDataset = () => {
+  const alias = forgeAlias('reqGetDataset');
+  cy.intercept({ method: 'GET', url: API_REGEX.DATASET, times: 1 }, (req) => {
+    if (!stub.isEnabledFor('GET_DATASETS')) return;
+    const datasetId = req.url.match(API_REGEX.DATASET)[1];
```

Comment:
As you made below, maybe add optional chaining operator `?.` before `[1] `

Reply:
I'm not sure it is really useful though, because it won't help much if the pattern of the dataset id is not detected in the interception url :slightly_smiling_face: